frb_example/benches/dart/lib/main.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:convert';
+
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+import 'package:flutter_rust_bridge_benchmark/bridge_definitions.dart';
+import './seeder.dart';
+import 'package:logging/logging.dart';
+import 'package:uuid/uuid.dart';
+import 'ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
+
+void main(List<String> args) async {
+  String dylibPath = args[0];
+  // on web, env useJSON is unset, but server will send a --json cli arg instead
+  // this is because WASM cannot be fed any env var, but happily takes cli args.
+  final allowJSON = useJSON || args.contains('--json');
+  final log = Logger('frb_example/benches');
+  if (allowJSON) {
+    Logger.root.level = Level.INFO;
+  } else {
+    Logger.root.level = Level.ALL;
+  }
+  Logger.root.onRecord.listen((record) {
+    print('${record.level.name}: ${record.time}: ${record.message}');
+  });
+  log.fine('flutter_rust_bridge_benchmark_suite example program start (dylibPath=$dylibPath)');
+  log.finer('construct api');
+  if (isSilicon) {
+    log.warning("⚠️  [env] SILICON: uses open method for dynamic library");
+  }
+  if (useJSON) {
+    log.info("🗃️  [env] JSON: switch to json output");
+  }
+  if (isWeb && !allowJSON) {
+    log.warning("⚠️  on Web, Rust logs to stdout are redirected to browser's console (or simply choose JSON)");
+  }
+  final api = initializeBenchExternalLibrary(dylibPath, useJSON: allowJSON);
+
+  // here freely do as much setup as you need,
+  // since only calls to bridge-wired functions will record any metric
+  final List<UuidValue> thousandUuids = seed(UuidSeeder(), just_1000);
+  final List<String> thousandStrings = seed(StringSeeder(), just_1000);
+  final List<UuidValue> hundredThousandUuids = seed(UuidSeeder(), just_100_000);
+  final List<String> hundredThousandStrings = seed(StringSeeder(), just_100_000);
+
+  // real game starts here
+  // --- (benchmarks:start) (avoid any initialization or computation)
+  // let's get rid of sync first
+  // lest stdout's output be intermingled
+  // (otherwise feel free to open a PR for opentelemetry)
+  api.handleSyncBool(input: false, hint: 'bool');
```

Comment:
Benchmarking is challenging, partially because we must get rid of every single time-consuming things that does not belong to the code being benchmarked. For example, `hint` is usually null for normal users, so making it null may make it slightly more realistic and faster. But anyway this one is tiny.

Reply:
this is the only way to label the execution and be able to nicely display them afterwards.
otherwise if you run e.g. `handle_uuids` let's say with 1000 / 100_000 entries, then you need a way to "label" your function calls to be able to differentiate between them.

Reply:
Maybe put labels in a separate dart array, since the call index already "label"s them

Reply:
Anyway given discussions below, this is not required in *this* PR